library/oneview_storage_volume_attachment_facts.py

```diff
@@ -229,9 +228,9 @@ def __get_paths(self, attachments, options, facts):
             path_id_or_uri = paths_options.get('pathId') or paths_options.get('pathUri')
 
             if path_id_or_uri:
-                paths = [self.oneview_client.storage_volume_attachments.get_paths(attachment_uri, path_id_or_uri)]
+                paths = [self.resource_client.get_paths(attachment_uri, path_id_or_uri)]
```

Comment:
get_paths doesn't take two parameters. 
Also, get_paths should run on an object of the storage_volume attachment resource, so you don't have to pass attachment_uri